Library/Formula/qxt.rb

```diff
@@ -0,0 +1,239 @@
+class Qxt < Formula
+  desc "LibQxt is an extension library for Qt"
+  homepage "https://bitbucket.org/libqxt/libqxt/wiki/Home"
+
+  stable do
+    url "https://bitbucket.org/libqxt/libqxt/get/v0.6.2.tar.gz"
+    sha256 "582426d6c81958dd1ac5ca4a9d49807e0a2204b7343e7d49c6613bcb148b5cb8"
+    patch :DATA
+  end
+
+  head do
+    url "https://bitbucket.org/libqxt/libqxt.git"
+
+    patch :p1, <<EOS
+--- a/src/widgets/mac/qxtwindowsystem_mac.cpp
++++ b/src/widgets/mac/qxtwindowsystem_mac.cpp
+@@ -89,11 +89,7 @@ QString QxtWindowSystem::windowTitle(WId window)
+     // most of CoreGraphics private definitions ask for CGSValue as key but since
+     // converting strings to/from CGSValue was dropped in 10.5, I use CFString, which
+     // apparently also works.
+-    err = CGSGetWindowProperty(connection, window, (CGSValue)CFSTR("kCGSWindowTitle"), &windowTitle);
+-    if (err != noErr) return QString();
+-
+-    // this is UTF8 encoded
+-    return QCFString::toQString((CFStringRef)windowTitle);
++    return QString();
+ }
+EOS
```

Comment:
Similar problem to the previous patch. The OSX API has been removed recently.

http://stackoverflow.com/questions/11892171/compiling-libqxt-in-mac-osx